tests/garage/garage.spec.js

```diff
@@ -16,4 +17,15 @@ test.describe('Add cars to garage', () => {
         const removeCarPopup = await editCarPopup.clickRemoveCarBtn();
         await removeCarPopup.clickConfirmRemoveCarBtn();
     });
+
+    test.describe('Garage (network)', () => {
+        test('should display received user name from mock', async ({ profilePage }) => {
+
+            // const mockedUserName = USER_MOCK_RESPONSE.data.name + ' ' + USER_MOCK_RESPONSE.data.lastName;
+            const userName = await profilePage.getUserName();
+
+            // For component UI testing with mocks I'd use hardcoded values
+            expect(userName).toStrictEqual('Mock User');
```

Comment:
and if you have implemented my previous comment you have to get rid of hardcoded values